web/massastation/src/mirage/server.js

```diff
@@ -79,7 +111,24 @@ function mockServer(environment = ENV.DEV) {
         });
         return new Response(200, {});
       });
-
+      this.post('plugin-manager', (schema, request) => {
+        const sourceURL = request.queryParams.source;
+        const storePlugin = schema.stores.findBy({ url: sourceURL });
+        // use the parameters of the store plugin if it exists
+        // otherwise generate random values
+        const newPlugin = {
+          id: faker.number.int(),
+          status: 'Up',
+          updatable: false,
+          name: storePlugin?.name || faker.lorem.word(),
+          author: storePlugin?.author || faker.person.firstName(),
+          description: storePlugin?.description || faker.lorem.sentence(),
+          version: storePlugin?.version || faker.system.semver(),
+          home: storePlugin?.url || `/plugin/massalabs/${faker.lorem.word()}/`,
+        };
+        schema.plugins.create(newPlugin);
+        return newPlugin;
```

Comment:
I am not sure here https://github.com/massalabs/station/blob/main/api/swagger/server/restapi/resource/swagger.yml#L386 it returns a 204 no content
```suggestion
        return null;
```